main.js

```diff
@@ -956,6 +969,20 @@ inviteUser = function(req, res, token, inviteEmail, adminUserName, companyId) {
            mailer.sendMail(req.body.invite_email, 'Cloudbreak invite' , templateFile, {user: adminUserName,
            invite: process.env.SL_ADDRESS + '/account/register?token=' + userTempToken + '&email=' + inviteEmail + '&inviter=' + adminUserName})
            res.json({message: 'SUCCESS'})
+       } else if (createResp.statusCode == 409) {
+            getUserByName(req, res, token, inviteEmail, 'givenName,active', function(userData) {
+                if (userData.active == false) { // TODO: check verified instead
+                    var tempToken = userData.givenName;
+                    console.log(tempToken)
+                    var templateFile = path.join(__dirname,'templates','invite-email.jade')
+                    mailer.sendMail(req.body.invite_email, 'Cloudbreak invite' , templateFile, {user: adminUserName,
+                    invite: process.env.SL_ADDRESS + '/account/register?token=' + tempToken + '&email=' + inviteEmail + '&inviter=' + adminUserName})
+                    res.json({message: 'SUCCESS'})
+                } else {
+                    res.statusCode = 400
+                    res.json({message: 'Temporary registration failed. ' + createResp.body.message})
```

Comment:
From a user's perspective this is an invitation of user not a temporary registration.
The message should be a proper answer that could be presented on the UI too.